From fc83003f541ac690fe308c3f4ac36e62814a40db Mon Sep 17 00:00:00 2001
From: Adam Sampson <ats@offog.org>
Date: Wed, 9 Oct 2024 13:47:18 +0100
Subject: [PATCH] GUI: add missing include for Qt 6.8

This file uses qDebug, but didn't include <QDebug>, and it's no longer
transitively included with Qt 6.8.0.
---
 src/mkvtoolnix-gui/util/media_player.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mkvtoolnix-gui/util/media_player.cpp b/src/mkvtoolnix-gui/util/media_player.cpp
index d59a64e4a0..290fb492d6 100644
--- a/src/mkvtoolnix-gui/util/media_player.cpp
+++ b/src/mkvtoolnix-gui/util/media_player.cpp
@@ -2,6 +2,7 @@
 
 #include <Qt>
 #include <QAudioOutput>
+#include <QDebug>
 #include <QMediaPlayer>
 
 #include "mkvtoolnix-gui/util/media_player.h"
-- 
GitLab

